hostthedocs/util.py

```diff
@@ -76,28 +73,31 @@ class FileExpander(object):
     - tar
     """
 
+    ZIP_EXTENSIONS = ('.zip', )
+    TAR_EXTENSIONS = ('.tar', '.tgz', '.tar.gz', '.tar.bz2')
+
     def __init__(self, uploaded_file):
         self._file = uploaded_file
         self._handle = None
 
-    @staticmethod
-    def detect_compression_method(extension):
+    @classmethod
+    def detect_compression_method(cls, filename):
         """
         Attempt to detect the compression method from the filename extension.
 
         :param str extension: The file extension.
         :return: The compression method ('zip' or 'tar').
         :raises ValueError: If fails to detect the compression method.
         """
-        if extension == '.zip':
+        if any(filename.endswith(ext) for ext in cls.ZIP_EXTENSIONS):
```

Comment:
str.endswith and any are present in Python 2.6.
However, Python 2.6 (and even 3.3) is very old. I think their support could be removed.